charts/k8ssandra/values.yaml

```diff
@@ -455,8 +455,12 @@ reaper:
     traefik:
       # -- Traefik entrypoint where traffic is sourced. See https://doc.traefik.io/traefik/routing/entrypoints/
       entrypoint: web
-      # -- Tolerations to apply to the Reaper pods. See
-      # https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/ for background.
+  # -- Affinity to apply to the Stargate pods. See
```

Comment:
Very minor, but this should say Reaper not Stargate.

Reply:
Nice catch. I'll fix that.